homeassistant/components/xiaomi_aqara/binary_sensor.py

```diff
@@ -293,7 +293,7 @@ def parse_data(self, data, raw_data):
                 )
 
             if self.entity_id is not None:
-                self._hass.bus.fire(
+                self._hass.bus.async_fire(
```

Comment:
Side note: All of these events should be moved out of the binary sensor platform. Entities shouldn't fire custom events.

Reply:
I know (from the Xiaomi Miio PR) but that is out of scope for this PR.
The rest of that part of the code is untouched, I only use the appropriate async_fire instead of just fire.